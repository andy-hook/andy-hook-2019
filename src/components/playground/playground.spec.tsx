import React from "react"

import Playground from "./playground"
import { renderWithTheme } from "@test-utils"

jest.mock("@hooks/page-transition")

describe("<Playground />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <Playground />)
    expect(tree).toMatchSnapshot()
  })
})
