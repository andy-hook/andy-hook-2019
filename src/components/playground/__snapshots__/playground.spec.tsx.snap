// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Playground /> renders correctly 1`] = `
<div
  className="sc-EHOje PRAcs"
>
  <div
    className="sc-bdVaJa bUDcPf"
  >
    This is the playground page
  </div>
  <footer
    className="sc-ifAKCX hSmABr"
  >
    <div
      className="sc-bdVaJa bUDcPf"
    >
      <div
        className="sc-htpNat dzobEj"
      >
        I am a footer
      </div>
    </div>
  </footer>
</div>
`;
